integrations/maven-bloop/src/main/java/bloop/integrations/maven/BloopMojo.java

```diff
@@ -35,6 +37,21 @@
     @Parameter(property = "bloop.executionFork", defaultValue = "false")
     private boolean bloopExecutionFork;
 
+    @Parameter(property = "bloop.classpathOptions.bootLibrary", defaultValue = "true")
```

Comment:
Have you looked if the maven integration already supports all these?

Reply:
I've been through the plugin but I haven't found how this could be configured, so I assumed it didn't.

Reply:
Can you double-check?